### DISCUSSION TITLE
Add Play/Pause to tab context menu

### DISCUSSION THREAD

**Discussion by HOAMTH** [mbnuqw/sidebery#719] [Ideas]
I am in the process of completely replacing the native tabs with Sidebery and it'd be nice to implement tab-wise play/pause control to the context menu, essentially this: https://github.com/tigleym/menu-play-pause-tab

**Comment by mbnuqw** [comment]
Sidebery v5 (beta) already has the play/pause feature (middle click on audio badge), so it will be easy to create context menu items for this (placed this idea in todo). Plus, you can always use native context menu in Sidebery.
